tests/test_visitors/test_ast/test_loops/test_loops/test_await_in_loop.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from wemake_python_styleguide.violations.consistency import AwaitInLoopViolation
+from wemake_python_styleguide.visitors.ast.loops import WrongLoopVisitor
+
+code_that_breaks = """
+async def foo():
+    for _ in range(1):
+        await some()
+"""
```

Comment:
Don't forget to test:
- list comp `[await x for x in ...]`
- set comp
- dict comp
- gen exprs